### ISSUE TITLE
[email] pattern does not accept valid email

### ISSUE THREAD

**Issue by joona** [manvel-khnkoyan/jpv#8]
In the latest version, `[email]` pattern does not accept valid email address anymore.

```
error - the value of: {"email" = joona@foobar.fi} not matched with: "[email]"
```

**Comment by manvel-khnkoyan** [comment]
Thanks for reporting. 
The issue was fixed #9. Also, there are new useful futures

**Comment by manvel-khnkoyan** [comment]
Closed #9
